src/mappings.js

```diff
@@ -146,6 +146,8 @@ export const searchToQuery = args => {
   return Object.entries(args).reduce((query, [key, value]) => {
     if (!value) {
       return query;
+    } else if (value instanceof Date) {
```

Comment:
In which scenario will the date not be in the ISO 8601 format? I thought it was the default :)

Reply:
When the value is submitted to the server, it is converted from an ISO-8601 date to a JavaScript Date. When we pass this value to the query-string `stringify`, it gets converted to a date string that is not 8601, and therefore Bugzilla doesn't recognize. This has always been broken. I'll add a comment here for this.